batcher/aligned-sdk/src/sdk.rs

```diff
@@ -35,6 +35,18 @@ use futures_util::{
     StreamExt, TryStreamExt,
 };
 
+//Public constants for convenience
+pub const HOLESKY_PUBLIC_RPC_URL: &str = "https://ethereum-holesky-rpc.publicnode.com";
```

Comment:
this is used only for testing, it should be in the testing section at the end of the file